src/tests/ftest/util/daos_utils.py

```diff
@@ -388,15 +393,18 @@ def container_create_snap(self, pool, cont, snap_name=None, epoch=None,
             ("container", "create-snap"), pool=pool, svc=svc, cont=cont,
             sys_name=sys_name, snap=snap_name, epc=epoch, )
 
-    def container_destroy_snap(self, pool, cont, snap_name=None, epoch=None,
-                               svc=None, sys_name=None):
+    def container_destroy_snap(
+        self, pool, cont, epochs, snap_name=None, svc=None, sys_name=None):
```

Comment:
To support additional testing - like being able to test what happens when both an `epc` and `epcrange` are specified - I'd suggest the following.  I'm also taking the opportunity to align the variable names (epoch -> epc):
```suggestion
    def container_destroy_snap(self, pool, cont, snap_name=None, epc=None,
                               svc=None, sys_name=None, epcrange=None):
```

Reply:
Done. Thanks for the suggestion.